app.js

```diff
@@ -1,40 +1,35 @@
-const Discord = require('discord.js')
-const nbaUtil = require('./util/nbaUtils.js')
-const nbaData = require('./api/nbaData.js')
-// const ui = require('./response_ui/responseUI')
-const yUtils = require('./util/yahooUtils')
 // const yahooAuthData = require('./api/yahooAuthData')
 // const { getChickism } = require('./util/appUtils')
-
-const client = new Discord.Client()
 let refreshToken = ''
+const discord = require('discord.js')
+const { prefix } = require('./config.json')
+const fs = require('fs')
 
-client.on('message', async message => {
-  if (message.content === 'hi') {
-    message.channel.send('hellow world.')
-  }
+const client = new discord.Client()
+client.commands = new discord.Collection()
 
-  if (message.content === 'days') {
-    message.channel.send(nbaUtil.getDaysOfWeek(nbaUtil.getCurrentMonday()))
-  }
+const commandFiles = fs.readdirSync('./commands').filter(file => file.endsWith('.js'))
 
-  if (message.content === 'player') {
-    let firstName = 'Quinn'
-    let lastName = 'Cook'
-    nbaData.getPlayers().then(
-      resp => {
-        let teamId = nbaUtil.getTeamId(resp, firstName, lastName)
-        nbaData.getTeamSchedule(teamId).then(resp => console.log(resp))
-      }
-    )
-  }
+for (const file of commandFiles) {
+  const command = require(`./commands/${file}`)
+  client.commands.set(command.name, command)
+}
+
+client.once('ready', () => {
+  console.log('READY in apptest.js')
+})
+
+client.on('message', async message => {
+  if (!message.content.startsWith(prefix) || message.author.bot) return
+  const args = message.content.slice(prefix.length).split(/ +/)
+  const command = args.shift().toLowerCase()
+  if (!client.commands.has(command)) return
```

Comment:
If user enters an invalid command either create a !help command and refer to that or list all available commands.